config/internal/instascale/clusterrole.yaml.tmpl

```diff
@@ -21,6 +21,13 @@ rules:
   - delete
   - patch
 
+- apiGroups:
+  - ""
+  resources:
+    - secrets
+  verbs:
+    - get
```

Comment:
Could this be narrowed to specific resource names? Secrets are rather sensitive resources, so that's a best practice to avoid granting cluster-wide access whenever possible.

Reply:
That'd be perfect 👍🏼.

Reply:
I will update. Thank you!